From f2cb03d164d056a1b16d4fc218f3ff786b2194c8 Mon Sep 17 00:00:00 2001
From: Bradley Lowekamp <blowekamp@mail.nih.gov>
Date: Tue, 1 Jun 2021 19:52:45 +0000
Subject: [PATCH] Remove explicit instantiating of Image of deque of
 LabelObjectLine

This LabelObjectLine does not have an operator== defined required for
the same Image operator.
---
 Code/Explicit/src/sitkExplicitITKImage.cxx | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Code/Explicit/src/sitkExplicitITKImage.cxx b/Code/Explicit/src/sitkExplicitITKImage.cxx
index cde77666..24065fe5 100644
--- a/Code/Explicit/src/sitkExplicitITKImage.cxx
+++ b/Code/Explicit/src/sitkExplicitITKImage.cxx
@@ -60,8 +60,6 @@ template class SITKExplicit_EXPORT itk::Image<std::complex<double>, 2u>;
 template class SITKExplicit_EXPORT itk::Image<std::complex<double>, 3u>;
 template class SITKExplicit_EXPORT itk::Image<std::complex<float>, 2u>;
 template class SITKExplicit_EXPORT itk::Image<std::complex<float>, 3u>;
-template class SITKExplicit_EXPORT itk::Image<std::deque<itk::LabelObjectLine<2u>, std::allocator<itk::LabelObjectLine<2u> > >, 1u>;
-template class SITKExplicit_EXPORT itk::Image<std::deque<itk::LabelObjectLine<3u>, std::allocator<itk::LabelObjectLine<3u> > >, 2u>;
 template class SITKExplicit_EXPORT itk::Image<unsigned char, 1u>;
 template class SITKExplicit_EXPORT itk::Image<unsigned char, 2u>;
 template class SITKExplicit_EXPORT itk::Image<unsigned char, 3u>;
-- 
2.25.1

